addons/web_dashboard/models/res_users.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime, timedelta
+
+from openerp import models, api, _
+from openerp.exceptions import RedirectWarning
+from openerp.tools.misc import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class Users(models.Model):
+    _inherit = 'res.users'
+
+    def web_dashboard_get_users_by_state(self):
```

Comment:
@pga-odoo  Yes I will move it. But don't bother fixing the small issues I noticed, I'll fix them myself to be able to merge soon. If you can just answer my questions, it's ok :-)